ste/sender-blockBlobFromURL.go

```diff
@@ -64,10 +62,6 @@ func newURLToBlockBlobCopier(jptm IJobPartTransferMgr, destination string, pacer
 
 // Returns a chunk-func for blob copies
 func (c *urlToBlockBlobCopier) GenerateCopyFunc(id common.ChunkID, blockIndex int32, adjustedChunkSize int64, chunkIsWholeFile bool) chunkFunc {
```

Comment:
Can you add a comment somewhere here about how this optimization previously existed and we removed it for XXXX reason? That way in the future we can remember and noone will try to add it back again